cql3/expr/expression.hh

```diff
@@ -208,6 +209,14 @@ struct bind_variable {
     int32_t bind_index;
 };
 
+// A constant which does not yet have a date type. It is partially typed
+// (we know if it's floating or int) but not sized.
```

Comment:
So it's really strange that you call it an "untyped" constant, yet it does have a type, just not a size. So why not call it an unsized_constant?
Or at least partially_typed_constant?

Reply:
It does not have a type in our type system.